### ISSUE TITLE
About pth

### ISSUE THREAD

**Issue by XinyueZhangHust** [Ze-Yang/LGKD#2]
I greatly admire your outstanding contributions. I have encountered the following issues and would appreciate your response. The download links for the "15-5" and "15-1" pth files of VOC are the same. Should a new version be uploaded? Additionally, I have set up the same environment and attempted to load the file from the following link: https://github.com/Ze-Yang/LGKD/releases/download/v1.0/15-x-voc.pth. However, I encountered the error "PytorchStreamReader failed reading zip archive: failed finding central directory." Could this error be due to data loss during the upload process?

**Comment by Ze-Yang** [comment]
Thank you for your interest. In fact, we use the same pretrained weights for 15-1 and 15-5. That's why I name it as 15-x.

As for the weight link, I can sucessfully download it by a simple click. You may try to download it manually and upload to the server. I have not tested it with "wget xxx" on the server.

**Comment by XinyueZhangHust** [comment]
Thank you very much for your prompt response. I have successfully downloaded the 19-1.pth file for testing and saved the corresponding result images. However, I haven't been able to find the reason why I can't test with the 15-1 and 15-5 files yet. I will try again later.

**Comment by XinyueZhangHust** [comment]
I tried to train using lgkd_19-1.sh, but the mIoU for the twentieth class only reached 0.315. Is there anything wrong with my operation? I have loaded the provided pre-trained model during training.

**Comment by Ze-Yang** [comment]
Hi, you can try to obtain the PLOP's results first, either with PLOP's repo or our repo. This can make sure that your environment and configurations are correct.

**Comment by XinyueZhangHust** [comment]
Thank you for your response. I would like to ask, if I run the lgkd_15_5.sh script, which group of results from Table 1 of the paper will I obtain? Will running plop_15-5.sh provide the results of LGKD+PLOP?

**Comment by Ze-Yang** [comment]
You will get the results for LGKD+PLOP using `lgkd_15-5.sh` while PLOP using `plop_15-5.sh`.

**Comment by XinyueZhangHust** [comment]
Would you be able to share the trained 15-1.pth file or the corresponding VOC test results, please?

**Comment by Ze-Yang** [comment]
Please see the logs in [LGKD_15-1_log.zip](https://github.com/user-attachments/files/16725079/LGKD_15-1_log.zip) and the corresponding [model.pth](https://drive.google.com/file/d/1H6CWKgDN0XfS9fUnBPQBo5wgI_3T5c8Z/view?usp=drive_link). Note that these results are slightly different from the results reported in the paper as it is reproduced with one RTX4090 GPU. The server where the original model was stored malfunctioned now, I can provide the original model once it recovers.

**Comment by XinyueZhangHust** [comment]
Thanks for the helpful information. Best of luck in all your endeavors.
